src/components/scenes/CoinRankingScene.tsx

```diff
@@ -144,6 +146,12 @@ const CoinRankingComponent = (props: Props) => {
     }
   }, [])
 
+  React.useEffect(() => {
+    return navigation.addListener('focus', () => {
+      dispatch(checkEnabledExchanges())
+    })
+  }, [dispatch, navigation])
```

Comment:
Isn't there a `useFocusEffect` hook from react-navigation you could use?

Reply:
No, that's not appropriate for this use-case. There is a lot of extra logic in that hook that we don't need.